midend/predication.h

```diff
@@ -22,7 +22,7 @@ limitations under the License.
 
 namespace P4 {
 
-/**
+/*
```

Comment:
This docstring comment is intentional, using this form (or triple dashes) causes the tools to generate documentation for this function.